Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -994,6 +1015,8 @@ PhysicalParticleContainer::AddPlasma (int lev, RealBox part_realbox)
             amrex::ignore_unused(j,k);
 #endif
         });
+        // Need synchronize since we have tmp xlim, ylim, zlim arrays
+        amrex::Gpu::synchronize();
```

Comment:
is this needed?

Reply:
I don't think so - a streamSynchronize is implied by the below can to `ExclusiveScan`.

Reply:
thank you. I will remove it then